main.tf

```diff
@@ -118,3 +118,31 @@ module "ansible" {
   envs      = ["host=${aws_eip.default.public_ip}"]
   playbook  = "${var.ansible_playbook}"
 }
+
+# Restart dead or hung instance
+
+resource "null_resource" "check_alarm" {
+  triggers = {
+    action = "arn:aws:swf:${var.aws_region}:${var.aws_account_id}:${var.default_alarm_action}"
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "reboot" {
+  alarm_name          = "${module.label.id}-status-check-failed"
```

Comment:
this alarm_name should be `default` since everything is parameterized and it could mean anything.